Makefile.PL

```diff
@@ -45,7 +45,7 @@ my %WriteMakefileArgs = (
     "Test::More" => "0.98",
     "Test::NoTabs" => 0
   },
-  "VERSION" => "0.04",
+  "VERSION" => "0.05",
```

Comment:
Just out of curiosity is this auto generated because we don't seem to change anything in the `WriteMakefileArgs` but still updated the version?

Reply:
they are change through `DZIL` command